Wrappers.h

```diff
@@ -0,0 +1,47 @@
+#pragma once
+
+#include "BurgerShack.h"
+#include "FishingTrawler.h"
+#include "SteamLocomotive.h"
+#include "BurgerChain.h"
+#include "FishingFleet.h"
```

Comment:
Don't include what you don't use
You are not **using** any of these types in this header file, so don't include them. 
You **ARE** using them in the cpp file, so that's where you need to include them.
- [ ] revise